hcert_spec.md

```diff
@@ -217,23 +217,21 @@ A proposed payload schema for [EU Health Certficate v1](https://github.com/ehn-d
 
 # Appendix B - Trust management
 
-The signature of the HCERT requires a public key to verify. Countries, or institutions within countries, need to make these public keys available. Ultimately, every verifier needs to have a list of the public keys it is willing to trust (the public key is not part of the HCERT).
+The signature of the HCERT requires a public key to verify. Countries, or institutions within countries, need to make these public keys available. Ultimately, every Verifier needs to have a list of the public keys it is willing to trust (the public key is not part of the HCERT).
 
-A simplified variation on the ICAO "_Master list_" will be used, tailored to this health certificate application, whereby each country is ultimately responsible for compiling their own master list. The aid of a coordinating Secretariat for operational and practical purposes will be available.
+A simplified variation on the ICAO "_Master list_" will be used, tailored to this health certificate application, whereby each country is ultimately responsible for compiling their own master list and making that available to the other Participants. The aid of a coordinating Secretariat for operational and practical purposes will be available.
 
 The system consists of (only) two layers; for each Member State one or more country level certificate that each sign one or more document signing certificates that are used in day to day operations.
 
-The Member State certificates are called Certificate Signer Certificate Authorities (CSCAs) and are (typically) self-signed certificates. Countries may have more than one (e.g. in case of regional devolution). These CSCA certificates regularly sign the Document Signing Certificates (DSCs) used in day to day operations. 
+The Member State certificates are called Certificate Signer Certificate Authorities (CSCAs) and are (typically) self-signed certificates. Countries may have more than one (e.g. in case of regional devolution). These CSCA certificates regularly sign the Document Signing Certificates (DSCs) used for signing HCERTs. Member States will each maintain a public register of the DSC certificates that is kept current and published at a stable URL which is communicated to the Secretariat. Member States MUST remove any revoked or stale certificates from this list.
 
-These CSCA certificates regularly sign the Document Signing Certificates (DSC) used in day to day operations. Memberstates will each will maintain a public register of the DSC certificates that is kept current at a stable URL communicated to the secretariat.  Memberstates *must* remove any revoked certificates from this list.
+The Secretariat will regularly retrieve the Member States DSCs, verify these agains the list of CSCA certificates (which have been conveyed and verified by other means) and publish an aggregated list of DSCs in various formats for Verifiers to retrieve. 
 
```

Comment:
According to the DGC Architecture document, this will be handled by an upload process (push) rather than a retrieve (pull) process from the central authority. Should this specification be specific about what procedures to use for this exchange?

Reply:
This is a very a good point, we should focus on explaining the overarching principles rather than the details.

Reply:
Looks good.

Reply:
I've edited that sentence accordingly.